test/conformance/runtime/header_test.go

```diff
@@ -124,6 +124,50 @@ func (c *checkIPList) String() string {
 	return c.expected
 }
 
+type checkForwardedHeader struct {
+	expected string
+}
+
+// token as defined in https://tools.ietf.org/html/rfc7230#section-3.2.6
+var tokenMatcher = regexp.MustCompile(`^[0-9a-zA-Z!#$%&'*+-.^_|~]+$`)
+
+// approximation of quoted-string as defined in https://tools.ietf.org/html/rfc7230#section-3.2.6
+var quotedStringMatcher = regexp.MustCompile(`^"[^"]+"$`)
+
+func isDelimiter(r rune) bool {
+	return r == ';' || r == ','
+}
+
+// MatchString returns true if the passed string contains a roughly valid Forwarded header content.
+func (*checkForwardedHeader) MatchString(s string) bool {
+	for _, pair := range strings.FieldsFunc(s, isDelimiter) {
+		// Allow for a trailing delimiter. Some routers unfortunately do that.
+		if pair == "" {
+			continue
+		}
+		parts := strings.Split(strings.TrimSpace(pair), "=")
```

Comment:
what if it's `a=b=c`? `SplitN`, may be? Since this will pass token matcher validation.

Reply:
Will it? = is not an allowed character in token. I‘ll habe another look though

Reply:
I guess, it's a delimeter.